### PR TITLE
Moved AddUser from modal to screen

### PR DESCRIPTION
**Pull request by jinapurapu**
**How it looks:**
Screenshot from 2022-04-19 14-06-56

Closes https://github.com/miniohq/engineering/issues/726

### CODE REVIEW

**Review by oscarocastellanos — CHANGES_REQUESTED** [review]
The assign groups search box should have more space on top so it doesn't clash with the previous box. 

Icons in the help bar should be smaller and just one color. Please let me know if you need me to adjust the icons.

Is it possible to use the side by side help box format as @prakashsvmx has been doing in other recent screens?

**Comment by dvaldivia** [comment]
I think it's just some extra alignment 

image

**Review by dvaldivia — CHANGES_REQUESTED** [review]
There's a couple of issues, on error we are still redirecting, so avoid redirecting on error, also when we successfully ad we don't refresh the list of users, that might be a bug on user list screen


Screen Shot 2022-04-19 at 3 08 32 PM
Screen Shot 2022-04-19 at 3 08 14 PM

**Comment by dvaldivia** [comment]
Can you align like this @jinapurapu ?
Screen Shot 2022-04-19 at 4 09 14 PM

File: portal-ui/src/screens/Console/Users/AddUserScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+              </Button>
+           </Grid>
+           </Grid>
+           </form>
+           </Box>
+           </Grid>
+           <Grid xs={4}>
+           <Box>
+           <AddUserHelpBox />
+        </Box>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/AddUserScreen.tsx:283]: Please run prettier on this file to fix indentation

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Users/ListUsers.tsx [hunk 2]
```diff
@@ -389,8 +388,7 @@ const ListUsers = ({ classes, setErrorSnackMessage, history }: IUsersProps) => {
                             To get started,{" "}
                             <AButton
                               onClick={() => {
-                                setAddScreenOpen(true);
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Users/ListUsers.tsx:392]: `closeAddModalAndRefresh`   - function
`addScreenOpen`  - state
`AddUser` Component and Usage

please remove/cleanup these as they are no longer used in `ListUsers` component.

**Review by prakashsvmx — DISMISSED** [review]
LGTM :+1:

File: portal-ui/src/common/SecureComponent/permissions.ts [hunk 3]
```diff
@@ -307,6 +308,8 @@ export const IAM_PAGES_PERMISSIONS = {
     IAM_SCOPES.ADMIN_DISABLE_USER,
     IAM_SCOPES.ADMIN_DELETE_USER,
   ],
+  [IAM_PAGES.USER_ADD]: [
+    IAM_SCOPES.ADMIN_CREATE_USER, // displays create user button
```
Comment (dvaldivia) [inline portal-ui/src/common/SecureComponent/permissions.ts:312]:
```suggestion
    IAM_SCOPES.ADMIN_CREATE_USER,], // displays create user button
```

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM.

**Review by oscarocastellanos — DISMISSED** [review]
Icon should  be correctly aligned and also the upper most field to the help box

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**